lib/collision.lua

```diff
@@ -26,10 +26,10 @@ function collision.collisionTest(a, b)
 		and b.collision_type == "rectangle" then
 		-- Rectangle VS Rectangle
 		local diffX = math.abs((a.x + a.width / 2) - (b.x + b.width / 2)) 
-						- (a.width + b.width) / 2
```

Comment:
Does the indentation changes in this file really make the code easier to read? Or why is this tile changed?

Reply:
No, I'll revert it.